sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/PartitionPumpManager.java

```diff
@@ -151,42 +144,27 @@ void startPartitionPump(PartitionOwnership claimedOwnership, Checkpoint checkpoi
                     partitionProcessor.processEvent(new EventContext(partitionContext, eventData, checkpointStore,
                         partitionEvent.getLastEnqueuedEventProperties()));
                     endProcessTracingSpan(processSpanContext, Signal.complete());
-                } catch (Exception ex) {
-                    /* event processing threw an exception */
-                    handleProcessingError(claimedOwnership, partitionProcessor, ex, partitionContext);
-                    endProcessTracingSpan(processSpanContext, Signal.error(ex));
+                } catch (Throwable throwable) {
+                    /* user code for event processing threw an exception - log and bubble up */
+                    endProcessTracingSpan(processSpanContext, Signal.error(throwable));
+                    throw logger.logExceptionAsError(new RuntimeException(throwable));
```

Comment:
Should we add a message to this RuntimeException before bubbling it up? Could be useful because often the first thing  I do is "Ctrl+F" for the message in code. There is another instance.

Reply:
Added message to both exceptions.